react-prebuild.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react';
+import {renderToString} from 'react-dom/server';
+import Logo from './kuma/javascript/src/Logo';
+
+let stdinAccumulator = '';
+
+process.stdin.on('data', input => {
+    stdinAccumulator += input.toString();
+});
+
+process.stdin.on('end', () => {
+    const locales = JSON.parse(stdinAccumulator);
+
+    const result = Object.keys(locales).reduce((acc, locale) => ({
+        [locale]: {
+            Logo: renderToString(<Logo
+                url={locales[locale].Logo.props.url}
```

Comment:
I suspect that this could be generalized to just `{...locales[locale].Logo.props}` so we just pass in whatever props are passed to us.